rest/model/task_annotations.go

```diff
@@ -203,3 +250,18 @@ func ValidateIssues(issues []APIIssueLink) error {
 	}
 	return catcher.Resolve()
 }
+
+func ValidateTaskLinks(links []APITaskLink) error {
```

Comment:
Can this be moved to the model/annotations package so it can be used by the UI also? Should have a function comment also.

Reply:
Refactored and moved to model package.